### DISCUSSION TITLE
Download

### DISCUSSION THREAD

**Discussion by exinReflex** [JustOptimize/return-ShowHiddenChannels#42] [Q&A] [answered]
how do i download it theres no button to download it

**Comment by exinReflex** [comment, marked as answer]
nvm i figured it out

**Comment by 1xSonny** [comment]
Can Someone tell me how to download it ??

**Reply by JustOptimize** [reply]
Follow the instructions in the readme
![image](https://user-images.githubusercontent.com/54294419/198609352-b37e0b1e-8282-4478-8106-dd4c76fb741f.png)
